src/util/highlight.js

```diff
@@ -0,0 +1,113 @@
+// copied from React InstantSearch
+const HIGHLIGHT_TAGS = {
+  highlightPreTag: `<ais-highlight-0000000000>`,
```

Comment:
Let’s use the one from InstantSearch.js: `__ais-highlight__` & `__/ais-highlight`. That way it’s not a breaking change & you can choose to escape the whole string without issue still.